app/core/RPCMethods/RPCMethodMiddleware.ts

```diff
@@ -131,8 +131,8 @@ export const checkActiveAccountAndChainId = async ({
       networkType && getAllNetworks().includes(networkType);
     let activeChainId;
 
-    if (isInitialNetwork) {
-      activeChainId = NetworksChainId[networkType];
+    if (isInitialNetwork && networkType !== RPC) {
```

Comment:
`isInitialNetwork` already excludes `RPC`. That seems to be the whole point of that variable - it represents whether the network is something other than RPC.
```suggestion
    if (isInitialNetwork) {
```

Reply:
OWWW, you are totally right it seems I missed the filter that was done on the `getAllNetworks()` function!
Totally unnecessary!